react/features/blur/middleware.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+import { MiddlewareRegistry } from '../base/redux';
+import { getLocalVideoTrack } from '../../features/base/tracks';
+import { getJitsiMeetGlobalNS } from '../base/util';
+
+import {
+    TOGGLE_BLUREFFECT
+} from './actionTypes';
+
+import {
+    blurIsEnabled,
+    blurIsDisabled
+} from './actions';
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+/**
+ * The redux middleware to handle enabling of the video blur updates in a React way.
+ *
+ * @param {Store} store - The redux store.
+ * @returns {Function}
+ */
+MiddlewareRegistry.register(({ getState, dispatch }) => next => action => {
+    switch (action.type) {
+    case TOGGLE_BLUREFFECT: {
```

Comment:
Why not convert `TOGGLE_BLUREFFECT` to async action? This way you can put the code from the middleware into the action creator. This way we can remove the middleware and the action type. WDYT?

Reply:
Removed middleware and created async action